docs/operator-manual/user-management/index.md

```diff
@@ -14,6 +14,9 @@ login history etc. So if you need such features it is strongly recommended to us
 
 !!! warning "Make sure to read about security limitations related to local users in [security considerations](../../security_considerations.md) document"
 
+!!! note
+    When you create local users, each of those users will need additional [RBAC rules](../rbac.md) set up, otherwise they will fall back to the `read-only` RBAC rule.
```

Comment:
```suggestion
    When you create local users, each of those users will need additional [RBAC rules](../rbac.md) set up, otherwise they will fall back to the default policy specified by `policy.default` field of the `argocd-rbac-cm` ConfigMap.
```

Reply:
Thanks, it's absolutely correct :+1: